.github/dependabot.yml

```
@@ -3,4 +3,21 @@ updates:
+      js-minor-updates:
+        patterns:
+          - "*"
+        update-types:
+          - "minor"
+          - "patch"
+    labels:
+      - "dependencies"
+      - "automated"
+    commit-message:
+      prefix: "deps"
+      include: "scope"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)